src/echo-common/tsconfig-esm.json

```diff
@@ -12,5 +12,5 @@
     "jsx": "react-jsx",
     "sourceMap": true
   },
-  "include": ["./src"]
+  "include": ["./src", "./test"]
```

Comment:
@kongsgaard this change is what is breaking the build.

When you add this folder, it changes the paths that the build generates (adds a `/src` and `/test`

If you update the package exports [here](https://github.com/PoeStack/poestack-sage/pull/63/files#diff-d317ec99621c5c9f43cf2cb01e13cac602d70c1cd732ee1d371331806defe4e7R16-R19) and [here](https://github.com/PoeStack/poestack-sage/pull/63/files#diff-d317ec99621c5c9f43cf2cb01e13cac602d70c1cd732ee1d371331806defe4e7R28-R32) with the new generated paths, everything works